src/registrar/assets/sass/_theme/_buttons.scss

```diff
@@ -211,6 +211,18 @@ a.usa-button--unstyled:visited {
     align-items: center;
 }
 
+.usa-button--justify-right {
```

Comment:
You do not need this class. USWDS has a helper class text-right that does just that (but see Zander's comment first)